cts/cli/regression.rules.exp

```diff
@@ -91,16 +91,14 @@ No rule found with ID=blahblah
 * Passed: crm_rule       - Try to check a rule that doesn't exist
 =#=#=#= Begin test: Try to check a rule that doesn't exist, with XML output =#=#=#=
 <pacemaker-result api-version="X" request="crm_rule -c -r blahblah --output-as=xml">
-  <status code="105" message="No such object">
-    <errors>
-      <error>No rule found with ID=blahblah</error>
-    </errors>
-  </status>
```

Comment:
I hate losing detail in the XML format because we're pushing everyone to use it, and I don't want to give them a reason not to. We can call the same out->err()'s in the XML implementation

Reply:
Good point. At that point I might as well move the `out->err()` calls back to `eval_rule()` in `pcmk_rule.c` so that they printed for all formats. Then remove the default error from the `rule_check_default` formatter.